packages/components/src/Avatar/src/RichIconAvatarImage.tsx

```diff
@@ -76,12 +76,12 @@ function RichIconAvatarImage(props: RichIconAvatarImageProps, ref: ForwardedRef<
     });
 
     const renderProps = useRenderProps<RichIconAvatarImageRenderProps>({
-        ...props,
         className: classNames,
         style: mergedStyles,
         values: {
             isDisabled: isDisabled || false
-        }
+        },
+        ...props
```

Comment:
I'm not sure it makes sense in the useRenderProps function. @alexasselin008 , did you want it done here too?

Reply:
ya, it doens't make sense here